pkg/asset/manifests/additionaltrustbundleconfig.go

```diff
@@ -115,9 +115,11 @@ func parseCertificates(certificates string) (map[string]string, error) {
 		}
 
 		if cert.IsCA {
-			sb.WriteString(string(pem.EncodeToMemory(block)))
+			logrus.Warn("The additionalTrustBundle certificate is either not v3 or a certificate authority")
```

Comment:
hmm, why are logging when we know it is a CA?

Reply:
also i think since we are assuming that the bundle consists of multiple certs, i think we should warn which one, maybe include the fingerprint or something, to say this specific one.